diagnostics/matlab/private/PlotBins.m

```diff
@@ -102,7 +102,7 @@ function PlotBins(pinfo)
          axis tight
       end
 
-   case {'lorenz_96_2scale','simple_advection'}
+   case {'lorenz_96_2scale','simple_advection', 'lorenz_96_tracer_advection'}
```

Comment:
this is the private `PlotBins.m` Needs lorenz_96_tracer_advection added to the public `plot_bins.m` to run.

Reply:
added in 7350f089bc4a44a273d04a31b62abd6d3e2c01b